### ISSUE TITLE
Multi GPU loss don't converge as like as one GPU, why?

### ISSUE THREAD

**Issue by LeoWang** [HiKapok/SSD.TensorFlow#86]
I just modify the params according to the README.
![image](https://user-images.githubusercontent.com/181878/50755864-5363fe80-1295-11e9-8999-4aaa4e5c3c6d.png)

![image](https://user-images.githubusercontent.com/181878/50755713-d46ec600-1294-11e9-9fd0-09cac301741b.png)
Should I change the lr bigger or not?

**Comment by HiKapok** [comment]
@LeoWang you are highly recommended to use pre-trained model when it's possible, [more here](https://arxiv.org/abs/1811.08883)

**Comment by jamiechoi1995** [comment]
I met the same problem with multi GPU.
I use pretrained vgg model to init.

**Comment by chengchen666** [comment]
Hi, I'm wondering how did you guys set up for the multi GPU training. I couldn't train on multi-gpu environments. It keeps giving shaping error.
